### ISSUE TITLE
Printing window shows when running in intellj but doesnt popup when running from release files

### ISSUE THREAD

**Issue by Optirx** [DavBfr/dart_pdf#1409]
I built a desktop application that uses the printing library to make and print receipts , I have it open the print settings windows window when i click print , it works fine when i run the app from intellj but when i try to run it from the release or debug files using the exe file it doesnt show the print settings window like its missing some kind of permission or plugin to communicate with windows

**Comment by MimiKhan** [comment]
Same happening to me. It's working fine in debug mode but not in release mode. Have you found any solution? @Optirx ?

**Comment by DavBfr** [comment]
I don't reproduce it. Can one of you create a test project that shows the issue?

**Comment by MimiKhan** [comment]
Yes Sure. I'll create and let you know. Can you please provide your email? I will email you ASAP. THANK YOU. 
@DavBfr

**Comment by DavBfr** [comment]
dev.nfet.net@gmail.com

**Comment by DavBfr** [comment]
Seems to work for me with a Windows 11 machine.

**Comment by MimiKhan** [comment]
Have you tried building application and running release executable. ? Its the issue in release mode. Nit in debug or in IDE run

**Comment by MimiKhan** [comment]
Can you please upload the release folder to the repo I provided? I wanna check. Whats your flutter and windows version? Can you please provide flutter doctor summary?

**Comment by MimiKhan** [comment]
What CMake version you are using for building?

**Comment by DavBfr** [comment]
I don't have write access to your repo.

```
flutter doctor 
Doctor summary (to see all details, run flutter doctor -v):
[√] Flutter (Channel stable, 3.13.2, on Microsoft Windows [Version 10.0.22621.2134], locale en-US)
[√] Windows Version (Installed version of Windows is version 10 or higher)
[X] Android toolchain - develop for Android devices
[X] Chrome - develop for the web (Cannot find Chrome executable at .\Google\Chrome\Application\chrome.exe)
[√] Visual Studio - develop for Windows (Visual Studio Community 2022 17.7.3)
[!] Android Studio (not installed)
[√] VS Code, 64-bit edition (version 1.81.1)
[√] Connected device (2 available)
[√] Network resources
```

cmake version 3.27.4

**Comment by DavBfr** [comment]
I just tested with the printing master branch, all is fine on my side.
I have a slightly more recent version of Visual Studio (17.7.3) Other than that, I don't know.

Could you try to run in profile mode to see if there is a difference?

**Comment by MimiKhan** [comment]
maybe I should update Flutter. what do you say? I have currently 3.10.+ and yours 3.13.2

**Comment by DavBfr** [comment]
I tested 3.10.1 too with no issues.

**Comment by MimiKhan** [comment]
ohhh. then why not work from my end? I have all the same things as you.

**Comment by MimiKhan** [comment]
Can I send you my executable file with the required files and test it on your end? please.?
If you can test it on your system. I have tried on my 2 systems. not working here

**Comment by DavBfr** [comment]
yes

**Comment by MimiKhan** [comment]
Ohhh. Thank you so much. I ll upload to drive and send you link in email.

**Comment by MimiKhan** [comment]
I have sent you an email. Please check. Thank you ♥️

**Comment by MimiKhan** [comment]
Should I have to use both packages in the project?
printing & pdf? I have used both.
